packages/babel-plugin-transform-es2015-parameters/src/rest.js

```diff
@@ -30,6 +30,8 @@ let memberExpressionOptimisationVisitor = {
   },
 
   Flow(path) {
+    // Do not skip TypeCastExpressions as the contain valid non flow code
+    if (path.isTypeCastExpression()) return;
```

Comment:
Was wondering if there was anything else in the codebase with path.skip on Flow but it doesn't look like it